plugins/quota/validate.go

```diff
@@ -3,38 +3,65 @@ package quota
 import (
 	"fmt"
 
+	daemon "github.com/gluster/glusterd2/glusterd2/daemon"
 	"github.com/gluster/glusterd2/glusterd2/volume"
 	"github.com/gluster/glusterd2/glusterd2/xlator"
+	"github.com/gluster/glusterd2/pkg/errors"
 )
 
 const (
-	quotaEnabledKey = "features.quota"
+	quotaEnabledKey = "quota.enable"
 )
 
-func validateOptions(v *volume.Volinfo, key string, value string) error {
+func validateOptions(v *volume.Volinfo, key, value string) error {
 
-	if isQuotaEnabled(v) == false {
-		err := fmt.Errorf("Quota not enabled to set this value: '%s'", key)
-		return err
+	// Check if volume is started
+	if v.State != volume.VolStarted {
+		return errors.ErrVolNotStarted
 	}
 
 	switch key {
-	case "deem-statfs":
+	case "enable":
```

Comment:
The daemon start stop them self have the check for is daemon running(pidfile exists). so having it there wont be of much use as we will be doing a redundant work. and in case we have it there, we change the volfile and update it. if we have it this way we can avoid it. As it is also one type of validation, I think its better to have it in the validation itself.